src/core/server/base.js

```diff
@@ -199,7 +199,8 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
             .map((item) => asyncConnectLoadState[item].error)
             .filter((item) => item);
 
-          if (apiErrors.length === 1) {
+          const isTest = env === 'test';
```

Comment:
Because you define this only once and use it on the line below, I think:

```js
if (apiErrors.length === 1 && env !== 'test') {
```

would be better.

Though, checking for environment right in the server page seems odd.